### ISSUE TITLE
Add the detection to the RoiManager

### ISSUE THREAD

**Issue by romainGuiet** [PreibischLab/RS-FISH#4]
Hi  @team,
Thank you for releasing this tool!

Would it be possible to add an option to the GUI to add the detection to the RoiManager? 

An easy workaround is to use the interactive mode and at the histogram step to use "Image>Overlay>To Roi Manager", but it's not compatible with advanced mode (nor batch)
Another workaround would be to make a script to go through the result table and make rois from coordinates, but it sounds like re-doing something.

Thank you again,

Best,

R

**Comment by StephanPreibisch** [comment]
Hi @romainGuiet, I am trying right now. Is there a convenience method for setting a ROI at slice z? It is not clear to me how to set 3D positions, specifically how to add a PointROI for a specific slice. Thanks so much!

**Comment by StephanPreibisch** [comment]
This does something, but I cannot go through the image and the spots pop up in their respective slices. However, it works to select them in the list. Any help would be great :)

https://github.com/PreibischLab/RadialSymmetryLocalization/commit/bab2883c5544265c6b53aff531aa8a17ae9c2749

Thanks so much!

**Comment by romainGuiet** [comment]
Hi @StephanPreibisch ,

>It is not clear to me how to set 3D positions...

I think it's clear for no one 😏 ! Main issue to my understanding is that it behaves differently if the image is a simple stack, or a hyperstack. [Please find here a discussion](https://forum.image.sc/t/rois-to-trackmate/48918/12) about it.

**Comment by StephanPreibisch** [comment]
So you think the current version as I wrote it is what users expect? It is already deployed to the update site if you want check ...

**Comment by romainGuiet** [comment]
I just tested bab2883 (cloned the repo) on the provided multi-dots.tif and I can get the ROIs in the RoiManager!

What I did next was to use "sort" in the RoiManager to have the ROIs ordered by slice (otherwise while reviewing ROIs, from one to another, user jumps in the stack to very different slices).

Would it slow down a lot the processing to add the ROI name (as shown in the RoiManager) to the Results Table ? This might facilitate the review process...

Thank you for the fast implementaion!
